deploy/rbac-csi-azurefile-controller.yaml

```diff
@@ -192,3 +192,26 @@ roleRef:
   kind: ClusterRole
   name: csi-azurefile-controller-secret-role
   apiGroup: rbac.authorization.k8s.io
+---
```

Comment:
the rbac setting here is actually not necessary since only driver on node requires the specific rbac

Reply:
I have made a change to fix the chart config issue and removed this unused rbac setting.